.yamllint

```diff
@@ -0,0 +1,6 @@
+---
+rules:
+  line-length:
+    max: 100
+    allow-non-breakable-words: true
+    allow-non-breakable-inline-mappings: false
```

Comment:
only the `max` line I wanted to change from the default. But the tool requires all fields to be specified when a given rule (here `line-length`) is specified. :man_facepalming: